### ISSUE TITLE
Crazy tabs when pasting

### ISSUE THREAD

**Issue by Kira-PH** [LeeTeng2001/better-glsl-plugin#5]
When I paste some code into a glsl file, the code gets tabbed out far enough to be off-screen.

**Comment by LeeTeng2001** [comment]
Yeah, i'm not experience enough to know how to solve the tabbing issue, it's like that when your glsl format is incorrect or when you paste in syntax that are currently not supported by glsl plugin. I'll try to add feature in the upcoming release, do you have an example file for me to test?

**Comment by mmontag** [comment]
+1 to this issue, it also results in the incorrect indent when you press enter for newline, and scrambles the indentation when you do the Reformat Code command. Good luck on the problem, maybe the C-like syntax will allow you to reuse some formatting code from another open-source plugin.
